module/solutions/helper.js

```diff
@@ -1970,6 +1970,85 @@ module.exports = class SolutionsHelper {
         return reject(error);
       }
     });
+  }
+
+  /**
+  * Update User District and Organisation In Solutions For Reporting.
+  * @method
+  * @name addReportInformationInSolution 
+  * @param {String} solutionId - solution id.
+  * @param {Object} userProfile - user profile details
+  * @returns {Object} Solution information.
+*/
+
+  static addReportInformationInSolution(solutionId,userProfile) {
+    return new Promise(async (resolve, reject) => {
+        try {
+
+            //check solution & userProfile is exist
+            if ( solutionId && Object.keys(userProfile).length > 0 ) {
```

Comment:
Instead of Object.keys(userProfile).length we can just check if what the keys we need is present or not simple.